src/menus/ContactDetailMenu.php

```diff
@@ -37,6 +43,25 @@ public function items()
             'url' => Url::toSearch('domain', ['contacts' => $this->model->id]),
             'visible' => Yii::getAlias('@domain', false) && (int) $this->model->used_count > 0,
         ];
+        $items = array_merge($items, [
+            [
+                'label' => AjaxModal::widget([
+                    'id' => 'set-templates-modal',
+                    'header' => Html::tag('h4', Yii::t('hipanel:finance', 'Set templates') . ': ' . Html::tag('b', "{$this->model->name} / {$this->model->organization}"), ['class' => 'modal-title']),
+                    'scenario' => 'set-templates',
+                    'actionUrl' => ['@requisite/bulk-set-templates', 'id' => $this->model->id],
+                    'size' => Modal::SIZE_LARGE,
+                    'toggleButton' => [
+                        'label' => '<i class="fa fa-fw fa-exchange"></i>' . Yii::t('hipanel:finance', 'Set templates'),
+                        'class' => 'clickable',
+                        'data-pjax' => 0,
+                        'tag' => 'a',
+                    ],
+                ]),
+                'encode' => false,
+                'visible' => $user->can('requisites.update'),
```

Comment:
This element should not be visible for contacts, that do not represent a requisite.